internal/report/output/privacy/privacy.go

```diff
@@ -68,9 +68,11 @@ const PLACEHOLDER_VALUE = "Unknown"
 func BuildCsvString(reportData *outputtypes.ReportData, config settings.Config) (*strings.Builder, error) {
 	csvStr := &strings.Builder{}
 	csvStr.WriteString("\nSubject,Data Types,Detection Count,Critical Risk Finding,High Risk Finding,Medium Risk Finding,Low Risk Finding,Rules Passed\n")
-	err := AddReportData(reportData, config)
```

Comment:
In which circumstances can it be nil?
I'm wondering if we shouldn't simply remove that `AddReportData` call from here. WDYT?

Reply:
No current circumstances. I added it as a precaution, because `BuildCsvString` is a public method and there's no guarantee that the `reportData` we pass to it will contain a privacy report. Myself, I'd be happier to keep the guard in, but perhaps this is paranoia on my part 😅